tools/importer/README.md

```diff
@@ -0,0 +1,50 @@
+## Porting various repositories into mbed-os
```

Comment:
Given that the script is called "importer.py" I recommend titling this section "Importing repositories into mbed-os"